### PR TITLE
Bump catapult/traceviewer to latest

### PR DESCRIPTION
**Pull request by paulirish**
This'll allow us to migrate to the shared FMP and TTI calculations

_edit_: Babel transpiling added to the build script, in order to address rampant use of destructuring. :)

### CODE REVIEW

File: lighthouse-core/third_party/traceviewer-js/LICENSE [hunk 1]
```diff
@@ -1,27 +0,0 @@
-Copyright 2015 The Chromium Authors. All rights reserved.
```
Comment (brendankenny) [inline lighthouse-core/third_party/traceviewer-js/LICENSE]: I think this file needs to be kept

**Comment by paulirish** [comment]
use strict injected. license restored.

**Comment by paulirish** [comment]
Failures in node 4 and 5:

```sh
/home/travis/build/GoogleChrome/lighthouse/lighthouse-core/third_party/traceviewer-js/value/histogram.js:493
      for (var [stat, option] of this.summaryOptions) {
               ^
SyntaxError: Unexpected token [
```

ugh. :/

**Comment by brendankenny** [comment]
yeah, hmm, even with `--harmony` that won't work in v4 or v5. I don't see much choice except patching random spots until we're all in on node 6 or upstreaming changes where they break older node.

**Comment by samccone** [comment]
>  I don't see much choice except patching random spots until we're all in on node 6 or upstreaming changes where they break older node.

we could also compile our code and deliver a bundle before npm publish and test runs.

--

I can open a CL for this

**Comment by brendankenny** [comment]
we're already running a conversion script for traceviewer. I'd rather just keep whatever we have to do in there

**Comment by samccone** [comment]
> we're already running a conversion script for traceviewer. I'd rather just keep whatever we have to do in there

https://github.com/GoogleChrome/lighthouse/blob/0295887b88eebf6d80146dd5b79ed6f018451d4b/lighthouse-core/scripts/build-traceviewer-module.js

Is more about converting a browser running system to work in node, and less about "compiling" javascript though.

**Comment by paulirish** [comment]
All good here. @brendankenny PTAL

**Comment by paulirish** [comment]
This PR updated catapult to 4959b8b832af4f0d91cf1b2bc9bed10fbcccad2e

**Pull request merged**